oidc-handler/core/pom.xml

```diff
@@ -0,0 +1,283 @@
+  specific language governing permissions and limitations
+  under the License.
+--><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+    <artifactId>org.apache.sling.oidchandler</artifactId>
+    <groupId>org.apache.sling</groupId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+    <groupId>org.apache.sling</groupId>
+    <artifactId>core</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>core</name>
+    <description>org.apache.sling - core</description>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
+                <version>3.5.0</version>
+                <executions>
+                    <execution>
+                        <id>run-bnd</id>
+                        <goals>
+                            <goal>bnd-process</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!-- Apache Felix SCR Plugin -->
+
+            <plugin>
+
+                <groupId>org.apache.felix</groupId>
+
+                <artifactId>maven-scr-plugin</artifactId>
+
+                <version>1.20.0</version>
+
+                <executions>
+
+                    <execution>
+
+                        <id>generate-scr-scrdescriptor</id>
+
+                        <goals>
+
+                            <goal>scr</goal>
+
+                        </goals>
+
+                        <configuration>
+
+                            <!-- Private service properties for all services. -->
+
+                            <properties>
+
+                                <service.vendor>Adobe</service.vendor>
+
+                            </properties>
+
+                        </configuration>
+
+                    </execution>
+
+                </executions>
+
+                <configuration>
+
+                    <outputDirectory>${project.build.directory}/classes</outputDirectory>
+
+                </configuration>
+
+                <dependencies>
+
+                    <dependency>
+
+                        <groupId>org.slf4j</groupId>
+
+                        <artifactId>slf4j-simple</artifactId>
+
+                        <version>1.5.11</version>
+
+                    </dependency>
+
+                </dependencies>
+
+            </plugin>
+
+
+
+            <!-- Apache Felix Bundle Plugin -->
+
+
+
+            <plugin>
+
+                <groupId>org.apache.felix</groupId>
+
+                <artifactId>maven-bundle-plugin</artifactId>
+
+                <version>2.5.3</version>
+
+                <inherited>true</inherited>
+
+            </plugin>
+
+
+            <plugin>
```

Comment:
If you inherit from the sling parent pom _and_ add a `bnd.bnd` file, this plugin configuration will not be necessary. A module with a `bnd.bnd` file is https://github.com/apache/sling-org-apache-sling-commons-html.